zulipterminal/config/keys.py

```diff
@@ -415,3 +426,19 @@ def commands_for_random_tips() -> List[KeyBinding]:
         for key_binding in KEY_BINDINGS.values()
         if not key_binding.get("excluded_from_random_tips", False)
     ]
+
+
+# Maps alternate nav keys to standard nav keys.
+# Refer urwid/command_map.py
+for cmd in STD_NAV_CMDS:
+    std_key, *alt_keys = keys_for_command(cmd)
+
+    for key in alt_keys:
+        if std_key == "end":
+            urwid_cmd = "cursor max right"
```

Comment:
We could import these constants from urwid instead.
```python3
...
CURSOR_MAX_RIGHT = 'cursor max right'
...
```

Reply:
That would only work for 2 cases ryt? It won't be consistent with the `else` conditional?

Reply:
We could have a dict that maps them accordingly. It would be more future proof since we would be importing from urwid directly.